### ISSUE TITLE
Error al levantar el compose:   standard_init_linux.go:211: exec user process caused "exec format error"

### ISSUE THREAD

**Issue by ddiaz2380** [pablokbs/plex-rpi#14]
hola Pablo, me da este error cuando levanto el compose

standard_init_linux.go:211: exec user process caused "exec format error"
ERROR: Service 'flexget' failed to build: The command '/bin/sh -c apk add --update tzdata python py-pip ca-certificates && pip install --no-cache-dir --disable-pip-version-check flexget==2.21.0 transmissionrpc==0.11 && rm -rf /var/cache/apk/*' returned a non-zero code: 1

**Comment by ConradoChiesa** [comment]
hola, yo estoy teniendo el mismo problema, quiero agregar que no lo estoy corriendo en una RB no se si eso tiene que ver.

Gracias y saludos.
Conrado.

**Comment by ddiaz2380** [comment]
Si basicamente ese el error, yo no lo estaba corriendo sobre la RB por eso
el error, un error basico pero bueno.

**Comment by pablokbs** [comment]
Claro, ese error es porque este repo solo funciona en raspberry (procesador ARM). Pero esa imagen seguro está para procesadores amd64: https://hub.docker.com/search?q=flexget&type=image

Habria que probar alguna de esas :)

**Comment by richardrey** [comment]
@pablokbs he creado una imagen multiplataforma que permite los amd64.

puedes usarla si quieres para actualizarlo, mi imagen esta en docker hub

rafito5889/flexget:3.1.67

un saludo
